services/optimiser/optimiser.py

```diff
@@ -27,138 +27,71 @@ def generate_model_string():
         @return:
         """
         model_str = """
```

Comment:
Is this in the correct branch?

Reply:
That was supposed to be the scheduler fix. I accidentally committed it. I have reverted it back.